### PR TITLE
BaiduNet8||CIFAR10 inference||Baidu-USA-GAIT

### PR DESCRIPTION
**Pull request by BAIDU-USA-GAIT-LEOPARD**
upload the CIFAR10 inference results yielded by Baidu USA

### CODE REVIEW

File: CIFAR10/inference/BAIDU-USA-GAIT_BaiduNet8_1GV100_pytorch.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+	    "version": "v1.0",
+	    "author": "Baidu USA GAIT LEOPARD team: Baopu Li, Zhiyu Cheng, Jiazhuo Wang, Haofeng Kou, Yingze Bao", 
+	    "authorEmail": "baopuli@baidu.com, zhiyucheng@baidu.com, jiazhuowang@baidu.com, haofengkou@baidu.com, yingzebao@baidu.com",
+	    "framework": "PyTorch torch-1.0.1.post2-cp35-cp35m-linux_x86_64.whl",
+	    "codeURL": " https://github.com/BAIDU-USA-GAIT-LEOPARD/CIFAR10-Inference-BaiduNet8, commitHash: https://github.com/BAIDU-USA-GAIT-LEOPARD/CIFAR10-Inference-BaiduNet8/commit/8678e0bcc72e6b72d5eb66e997e1ae92f2eaf2a1",
```
Comment (codyaustun) [inline CIFAR10/inference/BAIDU-USA-GAIT_BaiduNet8_1GV100_pytorch.json]: Please split `commitHash` into a separate field or use the Github URL with the commit hash in it.

File: CIFAR10/inference/BAIDU-USA-GAIT_BaiduNet8_1GV100_pytorch.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
+{
+	    "version": "v1.0",
+	    "author": "Baidu USA GAIT LEOPARD team: Baopu Li, Zhiyu Cheng, Jiazhuo Wang, Haofeng Kou, Yingze Bao", 
+	    "authorEmail": "baopuli@baidu.com, zhiyucheng@baidu.com, jiazhuowang@baidu.com, haofengkou@baidu.com, yingzebao@baidu.com",
+	    "framework": "PyTorch torch-1.0.1.post2-cp35-cp35m-linux_x86_64.whl",
+	    "codeURL": " https://github.com/BAIDU-USA-GAIT-LEOPARD/CIFAR10-Inference-BaiduNet8, commitHash: https://github.com/BAIDU-USA-GAIT-LEOPARD/CIFAR10-Inference-BaiduNet8/commit/8678e0bcc72e6b72d5eb66e997e1ae92f2eaf2a1",
+	    "model": "BaiduNet8 using PyTorch JIT in C++",
+	    "hardware": "Baidu Cloud Tesela V100*1/60 GB/12 CPU",
```
Comment (codyaustun) [inline CIFAR10/inference/BAIDU-USA-GAIT_BaiduNet8_1GV100_pytorch.json]: Is "Tesela" a typo of "Tesla"?

File: CIFAR10/inference/BAIDU-USA-GAIT_BaiduNet8_1GV100_pytorch.json [hunk 3]
```diff
@@ -0,0 +1,17 @@
+{
+	    "version": "v1.0",
+	    "author": "Baidu USA GAIT LEOPARD team: Baopu Li, Zhiyu Cheng, Jiazhuo Wang, Haofeng Kou, Yingze Bao", 
+	    "authorEmail": "baopuli@baidu.com, zhiyucheng@baidu.com, jiazhuowang@baidu.com, haofengkou@baidu.com, yingzebao@baidu.com",
+	    "framework": "PyTorch torch-1.0.1.post2-cp35-cp35m-linux_x86_64.whl",
+	    "codeURL": " https://github.com/BAIDU-USA-GAIT-LEOPARD/CIFAR10-Inference-BaiduNet8, commitHash: https://github.com/BAIDU-USA-GAIT-LEOPARD/CIFAR10-Inference-BaiduNet8/commit/8678e0bcc72e6b72d5eb66e997e1ae92f2eaf2a1",
+	    "model": "BaiduNet8 using PyTorch JIT in C++",
+	    "hardware": "Baidu Cloud Tesela V100*1/60 GB/12 CPU",
+	    "latency": 0.683,
+	    "cost": 2.078e-3,
```
Comment (codyaustun) [inline CIFAR10/inference/BAIDU-USA-GAIT_BaiduNet8_1GV100_pytorch.json]: I am not familiar with Baidu Cloud's pricing. Can you send me a link to the instance price for this machine?

File: CIFAR10/inference/BAIDU-USA-GAIT_BaiduNet8_1GV100_pytorch.json [hunk 4]
```diff
@@ -0,0 +1,17 @@
+{
+	    "version": "v1.0",
+	    "author": "Baidu USA GAIT LEOPARD team: Baopu Li, Zhiyu Cheng, Jiazhuo Wang, Haofeng Kou, Yingze Bao", 
+	    "authorEmail": "baopuli@baidu.com, zhiyucheng@baidu.com, jiazhuowang@baidu.com, haofengkou@baidu.com, yingzebao@baidu.com",
+	    "framework": "PyTorch torch-1.0.1.post2-cp35-cp35m-linux_x86_64.whl",
```
Comment (codyaustun) [inline CIFAR10/inference/BAIDU-USA-GAIT_BaiduNet8_1GV100_pytorch.json]: Thanks for the detailed version information. Could you truncate it to "PyTorch v1.0.1" for the `framework` field and add a `pytorchWheel` field under `misc` with "torch-1.0.1.post2-cp35-cp35m-linux_x86_64.whl"

```suggestion
	    "framework": "PyTorch v1.0.1",
```

File: CIFAR10/inference/BAIDU-USA-GAIT_BaiduNet8_1GV100_pytorch.json [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+	    "codeURL": " https://github.com/BAIDU-USA-GAIT-LEOPARD/CIFAR10-Inference-BaiduNet8, commitHash: https://github.com/BAIDU-USA-GAIT-LEOPARD/CIFAR10-Inference-BaiduNet8/commit/8678e0bcc72e6b72d5eb66e997e1ae92f2eaf2a1",
+	    "model": "BaiduNet8 using PyTorch JIT in C++",
+	    "hardware": "Baidu Cloud Tesela V100*1/60 GB/12 CPU",
+	    "latency": 0.683,
+	    "cost": 2.078e-3,
+	    "top1Accuracy": 94.32,
+	    "timestamp": "2019-04-12",
+	    "misc": {
+	        "CUDA version": "10.0",
+	        "CuDNN version": "7.3.1"
```
Comment (codyaustun) [inline CIFAR10/inference/BAIDU-USA-GAIT_BaiduNet8_1GV100_pytorch.json]:
```suggestion
	        "CuDNN version": "7.3.1",
	        "pytorchWheel": "torch-1.0.1.post2-cp35-cp35m-linux_x86_64.whl"
```

**Review by codyaustun — CHANGES_REQUESTED** [review]

File: CIFAR10/inference/BAIDU-USA-GAIT_BaiduNet8_1GV100_pytorch.json (hunk 2 above)
Comment (BAIDU-USA-GAIT-LEOPARD) [reply CIFAR10/inference/BAIDU-USA-GAIT_BaiduNet8_1GV100_pytorch.json]: Sorry for the typo, it should be Tesla.

File: CIFAR10/inference/BAIDU-USA-GAIT_BaiduNet8_1GV100_pytorch.json (hunk 3 above)
Comment (BAIDU-USA-GAIT-LEOPARD) [reply CIFAR10/inference/BAIDU-USA-GAIT_BaiduNet8_1GV100_pytorch.json]: https://cloud.baidu.com/doc/GPU/Pricing.html#.E5.8C.85.E5.B9.B4.E5.8C.85.E6.9C.88.E8.AE.A1.E8.B4.B9

It is in Chinese, and its price is 5,399 RMB/ month, so its corresponding price in USD is about 805.82

File: CIFAR10/inference/BAIDU-USA-GAIT_BaiduNet8_1GV100_pytorch.json (hunk 1 above)
Comment (BAIDU-USA-GAIT-LEOPARD) [reply CIFAR10/inference/BAIDU-USA-GAIT_BaiduNet8_1GV100_pytorch.json]: OK, please refer to the updated version, thanks.

File: CIFAR10/inference/BAIDU-USA-GAIT_BaiduNet8_1GV100_pytorch.json (hunk 4 above)
Comment (BAIDU-USA-GAIT-LEOPARD) [reply CIFAR10/inference/BAIDU-USA-GAIT_BaiduNet8_1GV100_pytorch.json]: yes, we truncated it as advised.

File: CIFAR10/inference/BAIDU-USA-GAIT_BaiduNet8_1GV100_pytorch.json (hunk 5 above)
Comment (BAIDU-USA-GAIT-LEOPARD) [reply CIFAR10/inference/BAIDU-USA-GAIT_BaiduNet8_1GV100_pytorch.json]: Followed your suggestion.

**Comment by BAIDU-USA-GAIT-LEOPARD** [comment]
We updated the json file as suggested, please further review it at your convenience, many thanks.

File: CIFAR10/inference/BAIDU-USA-GAIT_BaiduNet8_1GV100_pytorch.json [hunk 6]
```diff
@@ -0,0 +1,19 @@
+{
+	    "version": "v1.0",
+	    "author": "Baidu USA GAIT LEOPARD team: Baopu Li, Zhiyu Cheng, Jiazhuo Wang, Haofeng Kou, Yingze Bao", 
+	    "authorEmail": "baopuli@baidu.com, zhiyucheng@baidu.com, jiazhuowang@baidu.com, haofengkou@baidu.com, yingzebao@baidu.com",
+	    "framework": "PyTorch v1.0.1",
+	    "codeURL": " https://github.com/BAIDU-USA-GAIT-LEOPARD/CIFAR10-Inference-BaiduNet8",
+	    "commitHash": "https://github.com/BAIDU-USA-GAIT-LEOPARD/CIFAR10-Inference-BaiduNet8/commit/8678e0bcc72e6b72d5eb66e997e1ae92f2eaf2a1",
```
Comment (codyaustun) [inline CIFAR10/inference/BAIDU-USA-GAIT_BaiduNet8_1GV100_pytorch.json]:
```suggestion
	    "commitHash": "8678e0bcc72e6b72d5eb66e997e1ae92f2eaf2a1",
```

File: CIFAR10/inference/BAIDU-USA-GAIT_BaiduNet8_1GV100_pytorch.json [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+	    "version": "v1.0",
+	    "author": "Baidu USA GAIT LEOPARD team: Baopu Li, Zhiyu Cheng, Jiazhuo Wang, Haofeng Kou, Yingze Bao", 
+	    "authorEmail": "baopuli@baidu.com, zhiyucheng@baidu.com, jiazhuowang@baidu.com, haofengkou@baidu.com, yingzebao@baidu.com",
+	    "framework": "PyTorch v1.0.1",
+	    "codeURL": " https://github.com/BAIDU-USA-GAIT-LEOPARD/CIFAR10-Inference-BaiduNet8",
+	    "commitHash": "https://github.com/BAIDU-USA-GAIT-LEOPARD/CIFAR10-Inference-BaiduNet8/commit/8678e0bcc72e6b72d5eb66e997e1ae92f2eaf2a1",
+	    "model": "BaiduNet8 using PyTorch JIT in C++",
+	    "hardware": "Baidu Cloud Tesla V100*1/60 GB/12 CPU",
+	    "latency": 0.683,
+	    "cost": 2.078e-3,
```
Comment (codyaustun) [inline CIFAR10/inference/BAIDU-USA-GAIT_BaiduNet8_1GV100_pytorch.json]: This doesn't seem correct based on that price. I think you accidentally multiplied by 10,000. In my staging area, this currently shows up as $20.78 USD, which is much higher than any other result we have received.

```suggestion
	    "cost": 2.078e-7,
```

**Review by codyaustun — CHANGES_REQUESTED** [review]

File: CIFAR10/inference/BAIDU-USA-GAIT_BaiduNet8_1GV100_pytorch.json (hunk 6 above)
Comment (BAIDU-USA-GAIT-LEOPARD) [reply CIFAR10/inference/BAIDU-USA-GAIT_BaiduNet8_1GV100_pytorch.json]: Thanks for the correction, we modified as suggested.

File: CIFAR10/inference/BAIDU-USA-GAIT_BaiduNet8_1GV100_pytorch.json (hunk 7 above)
Comment (BAIDU-USA-GAIT-LEOPARD) [reply CIFAR10/inference/BAIDU-USA-GAIT_BaiduNet8_1GV100_pytorch.json]: Yes, previously we multiplied 10,000 for the whole data. We corrected it as suggested.

**Comment by BAIDU-USA-GAIT-LEOPARD** [comment]
We updated our jason result file as suggested, many thanks for the patient guidance.

**Review by codyaustun — APPROVED** [review]

**Comment by codyaustun** [comment]
Looks good to me

**Pull request merged**